regex.c

```diff
@@ -5484,12 +5482,12 @@ re_comp (s)
 
   if (!re_comp_buf.buffer)
     {
-      re_comp_buf.buffer = (unsigned char *) malloc (200);	/* __MEM_CHECKED__ */
+      re_comp_buf.buffer = malloc (200);	/* __MEM_CHECKED__ */
```

Comment:
well, I'm not sure what's the rationale about this, but the `malloc() __MEM_CHECKED__` calls have been made on purpose.